pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -282,15 +291,13 @@ func (c *MCRouteController) syncMCFlowsForAllCIImps(activeGW *mcv1alpha1.Gateway
 		return err
 	}
 
+	activeGWNochange := c.checkGateWayIPChange(activeGW)
```

Comment:
activeGWNoChange

But maybe rename it to "noActiveGWChange".

Reply:
renamed to `activeGWChanged`.